src/utilities/sticky-manager/sticky-manager.ts

```diff
@@ -25,26 +25,28 @@ interface StickyItem {
   ): void;
 }
 
+const SIXTY_FPS = 1000 / 60;
```

Comment:
does polaris have a constants file? might be good to move this there if it exists